pcdsdevices/happi/containers.py

```diff
@@ -666,6 +666,10 @@ class EnvironmentalMonitor(LCLSItem):
                        optional=True, enforce=str)
     ioc_base = EntryInfo('Base PV of the EK9000 IOC', optional=True,
                          enforce=str)
+    ioc_chan_num = EntryInfo('Channel number for the environmental monitor.',
+                            optional=True, enforce=str)
+    ioc_card_num = EntryInfo('Card number for the environmental monitor.',
+                            optional=True, enforce=str)
```

Comment:
the pre-commit flake8 error comes from the non-matching indentation here:
```suggestion
    ioc_chan_num = EntryInfo('Channel number for the environmental monitor.',
                             optional=True, enforce=str)
    ioc_card_num = EntryInfo('Card number for the environmental monitor.',
                             optional=True, enforce=str)
```